### PR TITLE
Me view all user profiles

### LINKED ISSUE #25: View All User Profiles
As an admin, I would like to see a list of all the User Profiles so that I can keep track of who is using the system.

**Given** the user is an admin in the Tabloid application  
**When** they select the `User Profiles` menu option  
**Then** they should be directed to the User Profiles list page  
**And** each user in the list should display the full name, the display name and the user type.
**And** the list should be in ordered alphabetically by user display name

### PR DESCRIPTION
**Pull request by MorganEro**
close https://github.com/NSS-Day-Cohort-59/hermes-heroes-tabloidfullstack/issues/25
Created GetAllUserPrfiles method
add it to UserProfileController and IUserProfileRepository
authorized the controller
added getAllUserProfiles components to the client folder
add User Profile to the NavBar
added a css module and made some css for the user profile page

### CODE REVIEW

File: Tabloid/Controllers/HelloController.cs [hunk 1]
```diff
@@ -3,6 +3,7 @@
 
 namespace Tabloid.Controllers
 {
+    [Authorize]
     [Route("api/[controller]")]
```
Comment (dalamcd) [inline Tabloid/Controllers/HelloController.cs:7]: You don't need to add the [Authorize] attribute here. This is a test controller that has two endpoints, one to test without auth and another one to test with auth. Adding the attribute to the whole class would defeat the purpose of that.

**Review by dalamcd — CHANGES_REQUESTED** [review]
I left one note on the HelloController, and there are a couple other issues.
1. Please remove package-lock.json and package.json from the commits.
2. Removing strict mode from index.js is an unrelated change and shouldn't be part of this pull request. Please make sure that you discuss with your team before making such a large change (if you haven't already).

File: Tabloid/Controllers/HelloController.cs (hunk 1 above)
Comment (MorganEro) [reply Tabloid/Controllers/HelloController.cs:7]: removed it. thanks

**Comment by MorganEro** [comment]
spoke with classmates and they are good with removing strickmode. We were thought in front end to remove it because it caused errors for us

**Pull request merged**